src/sidebars/search/search-controller.js

```diff
@@ -56,6 +84,12 @@
                             if (!stockAdded) {
                                 self.stocks.push(stock);
                             }
+
+                            // Change the selection pointer to be the selected stock, if it exists in the list
+                            // (otherwise, set to -1, which is acceptable as there is no selection yet)
+                            self.stockSelectionPointer = self.stocks.map(function(stockItem) {
+                                return stockItem.code;
+                            }).indexOf(self.selection());
                         },
```

Comment:
Ensure you have 'AAPL' as a favourite stock, and search for 'a'. Key down to a stock a few down from AAPL (which should be at the top). Switch to the favourites tab and select AAPL. Switch back to the search tab and you'll see AAPL is selected. Give the search bar the focus and press arrow-down. You'll see the selection will jump back down to where it was initially, and not continue from AAPL at the top.